frontend/src/components/navbar/Navbar.tsx

```diff
@@ -1,31 +1,67 @@
-import React, { useContext } from "react";
+import React, { useContext, useState } from "react";
 import { Link } from "react-router-dom";
 import { LoginContext } from "../login/LoginManager";
 import "./Navbar.scss";
+import Hamburger from "hamburger-react";
 
 export const Navbar: React.FunctionComponent = () => {
   const loginContext = useContext(LoginContext);
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  function toggleMenu() {
+    if (isExpanded) {
+      setIsExpanded(false);
+    }
+  }
 
   return (
     <nav className="navbar" role="navigation" aria-label="main navigation">
+      <div className="mobile-menu">
+        <Hamburger toggled={isExpanded} toggle={setIsExpanded} />
+      </div>
       <Link to="/">
         <img
           className="navbar__logo"
           src="/logo.png"
           alt="Whale Spotting logo"
         />
       </Link>
-      <div>
-        {!loginContext.isLoggedIn ? (
-          <div></div>
-        ) : (
-          <Link to="/">
-            <a className="button is-primary" onClick={loginContext.logOut}>
-              <strong>Logout</strong>
-            </a>
-          </Link>
-        )}
-      </div>
+      <ul className={`menu-items ${isExpanded ? "expanded" : ""}`}>
+        <Link to="/" onClick={() => toggleMenu()}>
+          <h1>WHALESPOTTING</h1>
```

Comment:
Is this being caps intentional, or a style choice?

If it's a style choice, you can actually (and, in fact, it's better practice to) use CSS to set it to display in all caps, using `text-transform: uppercase;`!

However, I'm not sure having only one navbar item in all caps is the best for visual continuity, nor am I too happy with using the name of the site instead of "Home" - could we switch to just that please?

```suggestion
          <h1>Home</h1>
```